src/Components/Form/Form.tsx

```diff
@@ -1,32 +1,51 @@
 import React from "react";
 import { StyleSheet, css } from "aphrodite";
 import CustomInput from "components/CustomInput/CustomInput";
+import { palette } from "styles/palette";
+import { typography } from "styles/typography";
+import { useTranslation } from "react-i18next";
 
 const Form = () => {
+  const { t } = useTranslation();
+
   return (
-    <div className={css(styles.wrapper)}>
-      <h2>Add new tasks</h2>
-      <form>
+    <div className={css(styles.formWrapper)}>
+      <h2 className={css(typography.logoFont, styles.formTitle)}>
```

Comment:
I think that you can spread typography.logoFont in your formTitle. It should be cleaner.